docs/diginstroom/aanlevering-van-sips.md

```diff
@@ -0,0 +1,167 @@
+**Let op**: zet het `or`\-gedeelte van je OR-id om in kleine letters! `OR-123abcd` moet dus `or-123abcd` worden.
+
+Een token vraag je aan via een POST-call naar het token-endpoint per omgeving:
+
+<table >
+  <colgroup>
+    <col style="width: 15%;">
+    <col style="width: 85%;">
+  </colgroup>
+  <tbody>
+    <tr>
+      <th>
+        <p style="text-align: center;"><strong>INT</strong></p>
+      </th>
+      <td>
+        <p>https://s3-int.do.viaa.be/_admin/manage/tenants/&lt;tenant&gt;/tokens</p>
+      </td>
+    </tr>
+    <tr>
+      <th>
+        <p style="text-align: center;"><strong>QAS</strong></p>
+      </th>
+      <td>
+        <p>https://s3-qas.viaa.be/_admin/manage/tenants/&lt;tenant&gt;/tokens</p>
+      </td>
+    </tr>
+    <tr>
+      <th>
+        <p style="text-align: center;"><strong>PRD</strong></p>
+      </th>
+      <td>
+        <p>https://s3.viaa.be/_admin/manage/tenants/&lt;tenant&gt;/tokens</p>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+{: .important }
+**Let op**: de INT-omgeving is niet publiek bereikbaar! Wens je hiervoor een token aan te vragen, neem dan contact op met meemoo via [support@meemoo.be](mailto:support@meemoo.be).
+
+Om voor een gebruiker een token aan te vragen, voer je een POST-call uit met de nodige metadata als headers naar het endpoint van de omgeving waarvoor je een token wil aanvragen.
+
+Een token aanvragen voor de QAS-omgeving zou er dan zo uitzien:
+
+```bash
+curl -X POST \
+    -H "X-User-Secret-Key-Meta: MyS3Secret" \
+    -H "X-User-Token-Expires-Meta: +10" \
+    -u "Jef Metdepet+or-123abcd" https://s3-qas.viaa.be/_admin/manage/tenants/OR-123abcd/tokens
+```
+
+Waarbij de headers volgende betekenis hebben:
+
+*   `X-User-Secret-Key-Meta`: dit is het vrij te kiezen _secret_ waarmee je samen met de token zult authenticeren (dit is dus verschillend van het paswoord dat je gebruikt om in te loggen).
+*   `X-User-Token-Expires-Meta`: dit is de levensduur van je token, uitgedrukt in dagen. Het is een goed idee deze levensduur zo kort mogelijk te zetten.
+*   Je gebruikt de gebruikersnaam zoals hierboven vermeld (`Jef Metdepet+or-123abcd`).
+*   In het endpoint gebruik je als `<tenant>` de OR-id van jouw organisatie (hier met `OR` in hoofdletters!).
+
+
+Vervolgens wordt er naar je paswoord gevraagd. Daarna krijg je een token terug in de volgende vorm:
+
+```json
+{
+  "token": "8b89d382891a33f8f05f875c7086f1f3",
+  "owner": "Jef Metdepet",
+  "scope": "+or-123abcd",
+  "expiration": "2024-01-10T12:39:08.877Z",
+  "creation": "2024-01-05T12:39:08.000Z",
+  "secret": "MyS3Secret"
+}
+```
+
+Met deze `token` en `secret` kan je nu verbinden met de S3-server en SIP's uploaden. Voor meer info over het werken met S3 of het opzetten van een integratie met S3, zie: [https://docs.aws.amazon.com/s3/index.html](https://docs.aws.amazon.com/s3/index.html).
+
+**Opgelet**
+
+Je moet altijd het meemoo-specifieke S3-endpoint configureren in je integratie of meegeven bij elke operatie. Dit zijn:
```

Comment:
Is dit duidelijk genoeg? Is het een idee om wat voorbeelden via `aws s3 CLI` toe te voegen? Bv.:
`aws s3 cp {filename} s3:/{bucket}/{filename} --endpoint https://s3-qas.viaa.be`

Reply:
Klopt. Code-voorbeelden zijn een goed idee. Als het kan wachten is het voor volgende week.

Reply:
> Als het kan wachten is het voor volgende week.

Ja, geen probleem. Ik zou het zelfs zo, zonder code-voorbeelden, durven openstellen. In eerste instantie is het specifiek voor één use-case met een korte feedbackloop. De effectieve S3-implementatie is ook nog niet voor volgende week maar ik zou graag het administratieve deel al achter de rug hebben.